docs/serving/autoscaling/target-burst-capacity.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Target burst capacity"
+linkTitle: "Target burst capacity"
+weight: 50
+type: "docs"
+---
+
+_Target burst capacity_ is a [global and per-revision](./global-vs-per-revision.md) integer setting.
+
+This value specifies how large of a _burst_ of traffic a Knative application can handle without buffering.
+
+If a burst of traffic is too large for the application to handle, the Knative _Activator_ will be placed in the request path to protect the application revision and optimize request load balancing.
+<!-- Question: is revision accurate here or is it protecting replicas of an app?-->
+<!-- TODO: docs about Activator?-->
```

Comment:
@vagababov

Reply:
Activator for most intents and purposes is detail of implementation here. So I am not sure we want to add more docs here.